pkg/policy/api/rule_validation.go

```diff
@@ -122,7 +122,7 @@ func (e *EgressRule) sanitize() error {
 	l3DependentL4Support := map[interface{}]bool{
 		"ToCIDR":      false,
 		"ToCIDRSet":   false,
-		"ToEndpoints": false,
+		"ToEndpoints": true,
```

Comment:
We can't really do this before the datapath is ready, right?

Reply:
Yes, this is true. I can close this in favor of #3372 which contains the datapath changes?